tls.go

```diff
@@ -0,0 +1,268 @@
+package zgrab2
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zcrypto/tls"
+)
+
+// Shared code for TLS scans.
```

Comment:
There's some actual syntax for including examples in documentation, which then get rendered special when you run `godoc` / on https://godoc.org. This might be worth investigating.

Reply:
Adding an item to my trello board to add godoc-friendly docs